src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -351,12 +355,21 @@ def add_photon_series(
             "OnePhotonSeries" not in metadata_copy["Ophys"]
         ), "Received metadata for 'OnePhotonSeries' but `photon_series_type` was not explicitly specified."
 
+    assert nwbfile_module_name in [
+        "acquisition",
+        "processing",
+    ], "'nwbfile_module_name' must be either 'acquisition' or 'processing'."
+
     # Tests if TwoPhotonSeries//OnePhotonSeries already exists in acquisition
     photon_series_kwargs = metadata_copy["Ophys"][photon_series_type][photon_series_index]
     photon_series_name = photon_series_kwargs["name"]
 
-    if photon_series_name in nwbfile.acquisition:
-        warn(f"{photon_series_name} already on nwbfile")
+    ophys = get_module(nwbfile, name="ophys")
```

Comment:
I would move this into the processing sub-level; note that this could otherwise result in the creation of an empty `ophys` processing module even if I had set the location to 'acquisition' (since the helper `get_module` returns if found and creates+attaches to nwbfile if not)

Can you add a test for that as well?

Reply:
good catch! I'll add a test too, thank you.